MAINTAINERS

```diff
@@ -1 +1,2 @@
 keatonwalker
```

Comment:
You should remove him or he'll be added to or reviews

Reply:
good call.  Thx (90s style)